src/SDKs/_metadata/datafactory_resource-manager.txt

```diff
@@ -1,11 +1,11 @@
-2018-05-14 07:31:34 UTC
+2018-05-22 18:36:15 UTC
 
 1) azure-rest-api-specs repository information
 GitHub user: Azure
 Branch:      master
-Commit:      eed831212786a36c6575fd6d20d2a970dc96cfa5
+Commit:      7a6afb0f255ea0cdbcb100b9f17ff670fda64c54
 
 2) AutoRest information
 Requested version: latest
-Bootstrapper version:    C:\Users\gadongji\AppData\Roaming\npm `-- autorest@2.0.4262  
+Bootstrapper version:    C:\Users\fgutierr\AppData\Roaming\npm +-- autorest@2.0.4262  `-- oad@0.1.7   `-- autorest@1.9.1  
```

Comment:
Please create a generate.ps1 script similar to [this](https://github.com/frankycrm14/azure-sdk-for-net/blob/a746c7e5b1213b6ae34dbb9ef18d49457459062a/src/SDKs/Compute/Management.Compute/generate.ps1) and regenerate the code.

Reply:
done